drivers/ieee802154/Kconfig.rf2xx

```diff
@@ -5,17 +5,20 @@
 
 menuconfig IEEE802154_RF2XX
 	bool "ATMEL RF2XX Driver support"
-	depends on NETWORKING
+	depends on NETWORKING && HAS_DTS
 	select SPI
 	select NET_L2_IEEE802154_SUB_GHZ
 
 if IEEE802154_RF2XX
 
-config IEEE802154_RF2XX_DRV_NAME
-	string "ATMEL RF2XX Driver's name"
-	default "IEEE802154_rf2xx"
-	help
-	  This option sets the driver name
+DT_IEEE802154_COMPAT_RF2XX := atmel,rf2xx
+
+config NET_CONFIG_IEEE802154_DEV_NAME
+	default "$(dt_compat_get_str, \
+		 label, \
+		 $(DT_IEEE802154_COMPAT_RF2XX), \
+		 0, \
+		 RF2XX_0)"
```

Comment:
Could maybe put on one line and add a comment. Gets a bit over 80 chars, but no catastrophe.

Thinking something like this:

```
config NET_CONFIG_IEEE802154_DEV_NAME
	# Fetch 'label' property from instance 0, defaulting to "RF2XX_0"
	default "$(dt_compat_get_str,label,$(DT_IEEE802154_COMPAT_RF2XX),0,RF2XX_0)"
```

Maybe those comments would get annoying if it gets used in lots of places, but might be nice as long as it's just used in a few places.

Reply:
yes, sure!